### ISSUE TITLE
Single text label for a MultiLineString

### ISSUE THREAD

**Issue by bejucoo** [maplibre/maplibre-gl-js#2693]
I'm trying to display text labels for MultiLineString features in a geoJSON file. I'm using the `symbol-placement: point `option so I can see the labels on different zoom levels and not just when I get really close as it would happen if I use the `line` option.

```
...

map.addSource('source_data', {
        type: 'geojson',
        data: 'sample.geojson'
    });

map.addLayer({
    "id": "symbols",
    "type": "symbol",
    "source": "source_data",
    "layout": {
        "symbol-placement": "point",
        "text-field": '{Name}',
        "text-size": 40,
        "text-justify": "center",
        "text-allow-overlap": false
      }
});

...
```

The thing is that when the MultiLinestring has more than one line, MapLibre is rendering a label for every one of them as seen in the image below. Would it be possible to render only one text label for each MultiLineString feature? 

I'm leaving a sample of the geoJSON file I'm using (with the two features of the image) because the original file is really long: https://www.jsonblob.com/1120054652224946176

![image](https://i.stack.imgur.com/YW2Md.jpg)

**Comment by HarelM** [comment]
I'm not sure it's possible for lines as well, I think it will be drawn once per tile (although I'm not 100% sure about it).
I think your best option is to use points to indicate where to place the label, but this might be just a workaround, not sure...

**Comment by HarelM** [comment]
There's also the `line-center` instead of `point` for `symbol-placement`.

**Comment by bejucoo** [comment]
Yes, at the end I could use points or try to simplify the features so they have only one line. Just wanted to see if it was possible. Thanks!
